appveyor.yml

```diff
@@ -8,8 +8,8 @@ environment:
   TOX_ENV: "pywin"
 
   matrix:
-    - PYTHON: "C:\\Python27"
-      PYTHON_VERSION: "2.7.8"
+    - PYTHON: "C:\\Python35"
+      PYTHON_VERSION: "3.5.2"
```

Comment:
The implication is actually that dbt 0.7.0 is only supported on Windows with py35/py36. You can't even import the snowflake connector on py27 because of some weird relative imports.